src/package/packageVersion.ts

```diff
@@ -333,6 +340,7 @@ export class PackageVersion {
     try {
       return await pollingClient.subscribe<PackageVersionCreateRequestResult>();
     } catch (err) {
+      const report = await this.getCreateStatus(createPackageVersionRequestId, connection);
```

Comment:
This looks like you're introducing a bug, unless `this.getCreateStatus` returns the same thing as `this.getCreateVersionReport`.  Why make 2 calls when there is an error?  Why not just save the first call and return that?

Reply:
@shetzel will check on the possible bug and I want to have most recent state given catch deals with timeout.